playground/components.go

```diff
@@ -440,7 +446,6 @@ func (r *RethEL) Run(svc *Service, ctx *ExContext) {
 			"--chain", "/data/genesis.json",
 			"--datadir", "/data_reth",
 			"--color", "never",
-			"--ipcpath", "/data_reth/reth.ipc",
```

Comment:
I do not remember why right now, but I think it was important to have this. Sorry for not documenting.